hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/audit/AuditProcessor.java

```diff
@@ -26,7 +26,11 @@
 public class AuditProcessor extends AbstractAuditPersistence implements Processor {
 
     private static Logger logger = LoggerFactory.getLogger(AuditProcessor.class);
-
+    
+    public enum Direction {
```

Comment:
This enum is for use with entities so can be in its own class in that package same as other enum used by entities.